index.js

```diff
@@ -1,9 +1,14 @@
 const { constants, hiscores } = require("osrs-api");
 const express = require('express')
+const path = require('path')
 const app = express()
-
 const port = 3000
 
+app.use(express.static(path.join(__dirname, 'public')));
+
+app.get('/', (req, res) => {
+  res.sendFile(path.join(__dirname + 'index.html'))
+});
 
 app.get('/:username', function(req, res, error){
```

Comment:
do you really need req, res and error?

Reply:
It looks like they're all needed
req - incoming function parameters
res - output/ result
error - error handling